cinch/roles/jenkins_slave/tasks/main.yml

```diff
@@ -36,42 +36,43 @@
     dest: "{{ jenkins_user_home }}/{{ jswarm_filename }}"
     url: "{{ jswarm_download }}"
 
-- name: create jswarm command
-  set_fact:
-    _jswarm_command: >
-      /usr/bin/java -Xmx2048m -jar {{ jenkins_user_home }}/{{ jswarm_filename }}
-      -master {{ jenkins_master_url }}
-      -name {{ jslave_name }}
-      -executors {{ jswarm_execs }}
-      -labels {{ jslave_label }}
-      -fsroot {{ jswarm_home }}
-      -mode exclusive
-      -disableSslVerification
-      -deleteExistingClients
+- name: upload swarm config file
+  template:
+    src: sysconfig_jenkins_swarm
+    dest: /etc/sysconfig/jenkins_swarm
+    owner: root
+    mode: 0644
+  notify: restart swarm
 
-- name: install swarm to rc.local
-  # TODO: replace with upstart file and systemd service
-  lineinfile:
-    dest: /etc/rc.d/rc.local
+- name: upload swarm systemd file
+  template:
+    src: swarm.service
+    dest: /usr/lib/systemd/system/swarm.service
```

Comment:
Should this go in **/etc/systemd/system** since the service is not provided by a package?